packages/flutter/lib/src/gestures/converter.dart

```diff
@@ -349,9 +348,32 @@ class PointerEventConverter {
                 pointer: state.pointer,
                 kind: kind,
                 device: datum.device,
+                position: state.lastPosition, // Change position in Hover
+                buttons: datum.buttons,
+                obscured: datum.obscured,
+                pressureMin: datum.pressureMin,
+                pressureMax: datum.pressureMax,
+                distance: datum.distance,
+                distanceMax: datum.distanceMax,
+                size: datum.size,
+                radiusMajor: radiusMajor,
+                radiusMinor: radiusMinor,
+                radiusMin: radiusMin,
+                radiusMax: radiusMax,
+                orientation: datum.orientation,
+                tilt: datum.tilt,
+              );
+            }
+            if (position != state.lastPosition) {
+              yield PointerHoverEvent(
+                timeStamp: timeStamp,
+                kind: kind,
+                device: datum.device,
                 position: position,
                 buttons: datum.buttons,
+                delta: state.deltaTo(position),
                 obscured: datum.obscured,
+                pressure: datum.pressure,
```

Comment:
pressure on a hover event? isn't that by definition zero?

Reply:
I agree it's weird, but `PointerHoverEvent` does accept pressure and actually passes it down to `super`. What's your suggestion?

Reply:
probably an error when we added that API. feel free to change it, it's very new. check the blame to see who added it just in case they had a good reason.

Reply:
It was added with https://github.com/flutter/flutter/pull/24554. Unfortunately the author has left Google and I can not ask if it was intentional. I've made a commit that deprecated the parameter.
https://github.com/flutter/flutter/pull/29998/commits/d89a99c503c0763aebd55c746b1e5207bbe97802

Reply:
Let's assume it wasn't intentional.

I think we can just remove it. No need to deprecate etc.

Reply:
It's done. https://github.com/flutter/flutter/pull/29998/commits/442234329a4d06d2e555891b7e0e8dea6cf31138